README.md

```diff
@@ -1 +1,7 @@
-# amld-linear-regression-example
\ No newline at end of file
+# VDK 
```

Comment:
As this is the first thing users would see when they open the repo, it needs to be more substantive. 

It should answer follow questions : 
* What is the purpose of the github repository 
* How to install it (e.g maybe click here)
* How to use it (maybe link to another readme.)